src/gov/nist/core/LexerCore.java

```diff
@@ -44,6 +44,7 @@
     public static final int START = 2048;
     public static final int END = START + 2048;
     // IMPORTANT -- This should be < END
+    public static final int IPV6 = END - 4;
```

Comment:
I dont have any idea on this, I just though that it's a value will be used as enum. Do you have any suggestion?